src/app/utils/concatClassNames.ts

```diff
@@ -0,0 +1,12 @@
+type ClassName = string | string[] | undefined | false | null;
+
+export default function concatClassNames(...classNames: ClassName[]): string {
```

Comment:
Any thoughts on using `classnames` package instead of our own?

It would save use from having to worry about stripping any issues with the name?

Reply:
We are trying not to rely too much on external packages so I think it works best for us to implement some of these trivial utils than using external packages